src/main/java/Group15/View/HomeScreenView.java

```diff
@@ -101,9 +100,16 @@ private static Pane createWorkoutsPane(String workoutsToShow) {
         title.setFont(Font.font("Arial", FontWeight.BOLD, 20));
         workoutsPane.getChildren().add(title);
 
-        // TODO: here we would fetch the workouts and put them into a list, but for now we just fetch mock data
-        MockWorkouts mockWorkouts = new MockWorkouts();
-        List<Workout> workouts = mockWorkouts.getWorkouts();
+        MockWorkouts mcokData = new MockWorkouts();
+        List<Workout> favoriteWorkouts = mcokData.getWorkouts();
+
+        List<Workout> workouts;
+        if ("Saved".equals(workoutsToShow)) {
+            workouts = WorkoutUtils.getSavedWorkouts();
+        } else {
```

Comment:
The method for this should probably be a ScrollPane instead of Pane, as one cannot see all the exercises otherwise